content/docs/beans/_index.md

```
@@ -1,41 +1,301 @@
+### Step 2 — Create the bean from the Data Source field
+
+1. In the dropdown, click **Create new bean**
+{{< image-sh src="13-create-new-bean-option.png" text="Create new bean option" >}}
+
+2. A modal dialog opens — fill in:
+   - **name** → `bookstoreDataSource`
+   - **type** → `org.postgresql.ds.PGSimpleDataSource`
+   {{< image-sh src="14-create-new-bean-modal.png" text="Create new bean modal" >}}
+
+3. Add the connection details under **properties**:
+   | Key | Value |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)